docs/sources/developer/architecture/components/metas.md

```diff
@@ -0,0 +1,105 @@
+# Metas
+
+Metas are key-value properties that are sent with each payload.
+
+Metas can be either static metas that once defined they are not changing across a session but they can be also dynamic
+metas that are either computed automatically for each payload or they are changed programatically by the end-user.
+
+## Available Metas
+
+### App
+
+The `app` meta ties signals with a specific app and it should not be changed across a session. It is required during
+initialization and it is the responsability of the end-user to define it.
+
+Properties:
+
+- `name` - the name of the app
+- `version` - the version of the app
+- `release` - a release identifier for the app, like a commit hash, a build number or a Docker container tag
+- `environment` - the environment where the app is running, like `staging` or `production`
+
+### Browser
+
+The `browser` meta helps with identifying the environment where the app is running and it should not change across a
+session. Altough it is not handled automatically by the core package, nor it is the responsability of the end-user to
+define it. Wrapper packages like `web-sdk` should handle it.
+
+Properties:
+
+- `name` - the name of the browser
+- `version` - the version of the browser
+- `os` - the operating system name and version where the browser is running
+- `mobile` - a flag indication if the browser is running on a mobile device
+
+### Page
+
+The `page` meta helps developers identify the page where a specific signal is coming from. It is changing automatically
+across a session and even though it is not handled automatically by the core package, wrapper packages like `web-sdk`
+should handle it. But unlike other metas, it can be also overwritten by the end-user if they have a custom mechanism for
+tracking the current page.
+
+Properties:
+
+- `url` - the URL of the current page
+- `id` - the name of the browser
+- `attributes` - a key-value object with additional attributes about the current page
+
+### SDK
+
+The `sdk` meta defines the following properties about the Faro library itself:
+
+- `name` - the name of the core library
+- `version` - the version of the library
+- `integrations` - the list of instrumentations that are used, identified with by the name and the version
+
+The `sdk` meta is handled internally by the core package and the end-user should not change it.
+
+### Session
+
+The `session` meta is a static meta that is used to link signals between them. It is not handled automatically by the
+core package and wrapper packages like `web-sdk` can handle it automatically. But unlike other metas, it can be also
+overwritten by the end-user if they have a different way of defining what a session in.
+
+Properties
+
+- `id` - the name of the browser
+- `started` - the date when the session started
```

Comment:
Q: is this a Date or a Timestamp?

Reply:
Actually... it doesn't exist in the API :-/. Gonna remove it from the docs. Need a separate PR to remove it from the interface.